extension/api.js

```diff
@@ -83,10 +88,20 @@
 			url += 'api/v3/notifications';
 		}
 
-		url += participating;
+		url += `?${query.join('&')}`;
 
 		xhr('GET', url, opts, (data, status, response) => {
 			const interval = Number(response.getResponseHeader('X-Poll-Interval'));
+			const linkheader = response.getResponseHeader('Link');
+
+			const getLinkPages = linkheader => {
+				for (const link of linkheader.split(', ')) {
+					if (link.endsWith('rel="last"')) {
+						return Number(link.slice(link.lastIndexOf('page=') + 5, link.lastIndexOf('>')));
```

Comment:
`find` with lambda function looks cryptic, but I can compile this, so let me try.